### PR TITLE
Update 03-generators.mdx

### PR DESCRIPTION
**Pull request by nikolasburk**
Added a few more community generators (+ also made the display of the community genewrators more consistent by naming the bullet exactly after the generator and adding a short description that explains what the generator does)

### CODE REVIEW

**Review by janpio — APPROVED** [review]
<3

**Comment by molebox** [comment]
Ahhh I didnt see this, sorry @nikolasburk ! Looks like the nexus one is the newest addition.

File: content/200-concepts/100-components/01-prisma-schema/03-generators.mdx [hunk 1]
```diff
@@ -64,8 +64,11 @@ The following is a list of community created generators.
...
-- [TypeGraphQL & Prisma integration](https://github.com/MichalLytek/typegraphql-prisma)
+- [`prisma-dbml-generator`](https://notiz.dev/blog/prisma-dbml-generator): Transforms the Prisma schema into [Database Markup Language
+#](https://www.dbml.org/home/) (DBML) which allows for an easy visual representation
+- [`prisma-docs-generator`](https://github.com/pantharshit00/prisma-docs-generator): Generates an individual API reference for Prisma Client
+- [`prisma-json-schema-generator`](https://github.com/valentinpalkovic/prisma-json-schema-generator): Transforms the Prisma schema in [JSON schema](https://json-schema.org/)
+- [`typegraphql-prisma`](https://github.com/MichalLytek/typegraphql-prisma#readme): Generates [TypeGraphQL](https://typegraphql.com/) CRUD resolvers for Prisma models 
+- [`nexus-prisma`](https://github.com/prisma/nexus-prisma/): Allows to project Prisma models to GraphQL via [GraphQL Nexus](https://nexusjs.org/docs/)
+- [`prisma-nestjs-graphql`](https://github.com/unlight/prisma-nestjs-graphql): Generate object types, inputs, args, etc. from prisma schema file for usage with @nestjs/graphql module.
```
Comment (nikolasburk) [inline content/200-concepts/100-components/01-prisma-schema/03-generators.mdx]:
```suggestion
- [`prisma-nestjs-graphql`](https://github.com/unlight/prisma-nestjs-graphql): Generates object types, inputs, args, etc. from the Prisma schema file for usage with `@nestjs/graphql` module
```

File: content/200-concepts/100-components/01-prisma-schema/03-generators.mdx [hunk 2]
```diff
@@ -64,8 +64,11 @@ The following is a list of community created generators.
 
 > **Note**: Note: Community projects are not maintained or officially supported by Prisma and some features may be out of sync. Use at your own discretion.
 
-- [DBML generator for Prisma](https://notiz.dev/blog/prisma-dbml-generator)
-- [Prisma documentation generator](https://github.com/pantharshit00/prisma-docs-generator)
-- [Prisma JSON Schema Generator](https://github.com/valentinpalkovic/prisma-json-schema-generator)
-- [Prisma Nestjs Graphql](https://github.com/unlight/prisma-nestjs-graphql)
-- [TypeGraphQL & Prisma integration](https://github.com/MichalLytek/typegraphql-prisma)
+- [`prisma-dbml-generator`](https://notiz.dev/blog/prisma-dbml-generator): Transforms the Prisma schema into [Database Markup Language
+#](https://www.dbml.org/home/) (DBML) which allows for an easy visual representation
```
Comment (nikolasburk) [inline content/200-concepts/100-components/01-prisma-schema/03-generators.mdx]:
```suggestion
- [`prisma-dbml-generator`](https://notiz.dev/blog/prisma-dbml-generator): Transforms the Prisma schema into [Database Markup Language](https://www.dbml.org/home/) (DBML) which allows for an easy visual representation
```

**Pull request merged**